modules/ztp-lvms-installing-lvms-cli.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp_far_edge/ztp-manual-install.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-lvms-installing-using-cli_{context}"]
+= Installing {lvms} by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install {lvms}.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the latest version of the {rh-rhacm} Operator.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create an `OperatorGroup` CR.
+
+.. Define the `OperatorGroup` CR and save the YAML file, for example, `lmvs-operatorgroup.yaml`:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: odf-lvm-operator-operatorgroup
+  namespace: openshift-storage
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "2"
+spec:
+  targetNamespaces:
+  - openshift-storage
+----
+
+.. Create the `OperatorGroup` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f lmvs-operatorgroup.yaml
+----
+
+. Create a `Subscription` CR:
+.. Define the `Subscription` CR and save the YAML file, for example, `lvms-subscription.yaml`:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: local-storage-operator
+  namespace: openshift-local-storage
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "2"
+spec:
+  channel: "stable"
+  name: local-storage-operator
```

Comment:
This is not correct, this is the LSO (name: local-storage-operator) not the LVMS. 
So:

spec:
  channel: stable-4.12
  name: lvms-operator